forage-android/src/main/java/com/joinforage/forage/android/network/data/CapturePaymentRepository.kt

```diff
@@ -123,14 +120,16 @@ internal class CapturePaymentRepository(
             }
 
             if (attempt == MAX_ATTEMPTS) {
-                logger.debug("Max attempts reached. Returning last response")
+                logger.e("Max polling attempts reached for capture request of Payment $paymentRef")
```

Comment:
include `contentId` in log? Also, worth including `paymentRef` in log tag / Context for easier DataDog search?